docs/UserGuide.md

```diff
@@ -598,11 +636,17 @@ Examples:
 
 ### **4.3 Navigating the UI**
 
-Although most of the buttons you see on the screen are clickable, the UI has been designed primarily to be navigated using the command line interface.
+Although most of the buttons you see on the screen are clickable, the UI has been designed primarily to be navigated using the command line interface. The commands listed in this section will help you navigate the UI without leaving keyboard.
+
+#### Viewing Today Panel : `today`
```

Comment:
Thinking of moving these  commands into their own individual section and create a section for today. While today section will only have one command, we can use that section to explain what today means  and what information is shown there.

Reply:
Good idea. I'll shift it around.